### PR TITLE
#49 Create a separate page for partner stores.

### PR DESCRIPTION
**Pull request by ushmita4**
I have created a separate page for partner brands. Consumers can check which brands we all collaborate with.

I have added the partner page to all the pages.
I have added the Partner Brand page as well.
 
Website:- https://ushmita4.github.io/Apni-Dukaan/
image
image

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
@ushmita4 properly align the navbar.

Write something else in place of check now and here are the brands and improve the heading design.
Arrange logos, side by side randomly instead of one below the other.
Use better images that don't have white or any other background.
Remove names of brands in bullet points, only logos will be fine.

**Comment by QAZIMAAZARSHAD** [comment]
I want something like this 

image

**Comment by ushmita4** [comment]
> I want something like this
> 
> image

Sir, I have changed. You can check it over the website.
Website:- https://ushmita4.github.io/Apni-Dukaan/


Regarding the navbar, it is showing well on my laptop. So, a suggestion from my side let somebody else take up the navbar issue.

Thank You. I learnt so much right now.

I hope it gets merged and if it still has some flaws let others take up the issue in a way to improvise it.

**Comment by ushmita4** [comment]
> > I want something like this
> > image
> 
> Sir, I have changed. You can check it over the website.
> Website:- https://ushmita4.github.io/Apni-Dukaan/
> 
> Regarding the navbar, it is showing well on my laptop. So, a suggestion from my side let somebody else take up the navbar issue.
> 
> Thank You. I learnt so much right now.
> 
> I hope it gets merged and if it still has some flaws let others take up the issue in a way to improvise it.

image
It's just a glimpse of the page.

**Comment by satyampgt4** [comment]
@ushmita4 try some CSS also...

**Comment by QAZIMAAZARSHAD** [comment]
Yes @ushmita4 apply some CSS.

Some logos are very large compared to others, 

And instead of here are our partners special collections just write "our partners" in the top center.
Add a few more brands also.

**Comment by QAZIMAAZARSHAD** [comment]
@ushmita4 feel free to reach @satyampgt4 if you face difficulty anywhere.

**Comment by ushmita4** [comment]
> @ushmita4 feel free to reach @satyampgt4 if you face difficulty anywhere.

Sir, I have added the changes.
Website:- https://ushmita4.github.io/Apni-Dukaan/Partner%20Stores.html

**Comment by QAZIMAAZARSHAD** [comment]
Properly align the navbar.

Remove the lines bordering the Our Partners heading.

"Keep supporting us and you will find more brands coming up in the near future" move this line in the center.

Add one more row of brands.

**Comment by ushmita4** [comment]
> Properly align the navbar.
> 
> Remove the lines bordering the Our Partners heading.
> 
> "Keep supporting us and you will find more brands coming up in the near future" move this line in the center.
> 
> Add one more row of brands.

Sir could you please check the changes.

Link to the page:- https://ushmita4.github.io/Apni-Dukaan/Partner%20Stores.html

**Comment by QAZIMAAZARSHAD** [comment]
Looks fine but there is more space in the right margin compared to the left margin.

Do something about it.

File: Partner Stores.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        <link rel="icon" href="./Images/favicon.ico" type="image/x-icon">
+        <style>
+          .partner-store-conatiner{
+​
+            display: flex;
+            flex-wrap: wrap;
+            align-items:center;
+            justify-content: center;
+            
+          }
```
Comment (satyampgt4) [inline Partner Stores.html:18]: remove new line and change it to
```
.partner-store-conatiner{
            display: flex;
            flex-wrap: wrap;
            align-items:center;
            justify-content: center;
          }
```

**Comment by ushmita4** [comment]
> Looks fine but there is more space in the right margin compared to the left margin.
> 
> Do something about it.

Sir I have tried to change. Kindly review it. Website:-https://ushmita4.github.io/Apni-Dukaan/Partner%20Stores.html

**Comment by QAZIMAAZARSHAD** [comment]
Fine.

But looks like your repo is not updated that's why it doesn't have a scroll to the top button, fetch upstream and add it to the partner's page also.

**Comment by ushmita4** [comment]
> Fine.
> 
> But looks like your repo is not updated that's why it doesn't have a scroll to the top button, fetch upstream and add it to the partner's page also.

Ok, Sir, let me try.

**Comment by QAZIMAAZARSHAD** [comment]
Always mention the issue number in the PR description like this #49.

**Comment by QAZIMAAZARSHAD** [comment]
Fill the google form also link in the slack channel.

**Pull request merged**